src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs

```diff
@@ -72,7 +72,8 @@ public virtual bool IsAttributeDefined(MemberInfo memberInfo, Type attributeType
         /// <returns>True if the specified attribute is defined on the type.</returns>
         public virtual bool IsAttributeDefined(Type type, Type attributeType, bool inherit)
         {
-            return this.IsAttributeDefined(type.GetTypeInfo(), attributeType, inherit);
+            var mi = (MemberInfo)type.GetTypeInfo();
```

Comment:
Consider renaming the variable to memberInfo

Reply:
Renamed